app/helpers/material_design/menu_helper.rb

```diff
@@ -0,0 +1,31 @@
+module MaterialDesign
+  module MenuHelper
+    def md_menu(menu_options: [], data: {}, form: nil, attribute: nil)
+      content_tag(:div, class: "menu", data: data) do
+        menu_options.each do |option|
+          concat(dropdown_menu_option(option, form, attribute))
```

Comment:
Eu preferiria que a gente removesse referências ao "dropdown"

```suggestion
          concat(menu_option(option, form, attribute))
```

Reply:
Digo isso porque o menu pode ser utilizado em action sheets, selects pesquisáveis, nested navigation, nem sempre é um contexto de "dropdown" somente

Reply:
Entendi, feito!